src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx

```diff
@@ -187,7 +202,7 @@ export const LabeledUserDropdown = ({
         {...rest}
       />
       {shouldShowWarningMessage && (
-        <StyledWarningMessageContainer>
+        <StyledWarningMessageContainer data-testid={`${rest['data-testid']}-warning-message`}>
```

Comment:
Just caught this issue while testing. It's not related to your changes, but I thought it worth fixing. It's actually related to lines 180-181 above:

```tsx
          return `${value.secretId}${value.nickname ? ` (${value.nickname})` : ''}${
            value.tag ? ` (${value.tag})` : ''
```

Tags should be getting passed through i18n – see how it's done in [`ParticipantTagChip.tsx`](https://github.com/ChildMindInstitute/mindlogger-admin/blob/feature/M2-6533-take-now-full-accounts/src/modules/Dashboard/components/ParticipantTagChip/ParticipantTagChip.tsx#L21).

Reply:
Done 👌